src/types/herokuTypes.ts

```diff
@@ -0,0 +1,118 @@
+interface HerokuIdentityProvider {
```

Comment:
Another discussion at https://lifeomic.slack.com/archives/CK65GRH0F/p1587567929368300 about avoiding creating types for third party APIs 😄 The goal would be to focus on declaring what entities are produced in converter function tests, avoid manual conversion and instead drive `createIntegrationEntity` toward more capabilities of auto-conversion.